### DISCUSSION TITLE
Redirect for not-found in middleware

### DISCUSSION THREAD

**Discussion by surnr** [vercel/next.js#62660] [App Router]
I want to redirect the user to the home page "/" if he goes to the not listed path which means "not-found.tsx" page will render, I can directly put a redirect in not-found page but is there a method to put a redirect in middleware instead of not-found. I think this will increase performance since we don't go 2 round network calls.

**Comment by mutasim77** [comment]
check this one : https://github.com/vercel/next.js/discussions/52233#discussioncomment-6779274
